data/inspector/components/actors-panel.js

```diff
@@ -24,21 +37,250 @@ var ActorsPanel = React.createClass({
 
   getInitialState: function() {
     return {
-      packets: this.props.packets,
-      selectedPacket: null
-    };
+      panelType: GLOBAL_ACTORS_POOLS
+    }
   },
 
   render: function() {
+    var { actors } = this.props;
+    var { panelType } = this.state;
+
+    var el;
+
+    switch(panelType) {
+    case GLOBAL_ACTORS_POOLS:
+      el = ActorsPools({ data: (actors && actors.global), key: 'global' });
+      break;
+    case TAB_ACTORS_POOLS:
+      el = ActorsPools({data: (actors && actors.tab), key: 'tab' });
+      break;
+    case ACTORS_FACTORIES:
+      el = ActorsFactories({
+        key: 'factories',
+        data: {
+          global: actors && actors.global,
+          tab: actors && actors.tab
+        }
+      });
+      break;
+    }
+
     return (
-      DIV({className: "actorsPanelBox"},
-        ActorsToolbar(),
-        DIV({}, "TODO")
+      DIV({ className: "actorsPanelBox" },
+        ActorsToolbar({ actions: this.props.actions,
+          currentPanelType: panelType,
+          panelTypesLabels: PanelTypesLabels,
+          onPanelTypeSelected: this.onPanelTypeSelected,
+        }),
+        DIV({ className: "actorsScrollBox" }, el)
       )
     );
+  },
+
+  onPanelTypeSelected: function(panelType) {
+    this.setState({
+      panelType: panelType
+    })
   }
 });
 
+/**
+ * @template This template renders 'ActorsPools' list in the tab content.
+ */
+var ActorsPools = React.createFactory(React.createClass({
+  /** @lends ActorsPools */
+
+  displayName: "ActorsPools",
+
+  render: function() {
+    var pool = [];
+
+    if (this.props.data) {
+      var { actorPool, extraPools } = this.props.data;
+      pool = pool.concat(actorPool, extraPools);
+    }
+
+    var poolTables = pool.map((poolData) => {
+      var pool = poolData.pool;
+      var poolId = poolData.id;
+      var actorClass = false;
+
+      if (!Array.isArray(pool)) {
+        pool = [pool];
+        actorClass = true;
+      }
+
+      if (pool.length > 0) {
+        return PoolTable({
+          pool: pool,
+          actorClass: actorClass,
+          id: poolId,
+          key: poolId,
+          searchFilter: this.state && this.state.searchFilter
```

Comment:
The search box is supported?
I don't see any filtering when typing into the search box (and having the Actors tab selected)

Honza